client/src/Component/MainTable.js

```diff
@@ -70,6 +73,7 @@ const MainTable = () => {
 					{clientData.map((client) => {
 						return (
 							<tr key={client.client_id}>
+								<td><Checkbox isChecked={props.checkedCheckboxes.includes(client.client_id)} label={client.name} onChange={(checked) => props.onHandleSelectedUserState(checked, client.client_id)} /></td>
```

Comment:
Can we add a `th` back in the table header? The header and body cells are misaligned:

<img src=https://user-images.githubusercontent.com/6434329/152359633-835cd4d0-8401-435f-9e56-72d6b8e9e699.png width=800>

Reply:
yikes, how did I not notice that!